src/Utils/testDataUtil.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.  
+ * Licensed under the MIT License.
+ */
+import * as CLM from '@conversationlearner/models'
+
+export function makeEntityValues(): CLM.MemoryValue[] {
+    return [{
+        "userText": "userText",
+        "displayText": "displayText",
+        "builtinType": null,
+        "resolution": {}
+    }]
+}
+
+export function makeFilledEntity(entityId?: string): CLM.FilledEntity {
+    return {
+        entityId: entityId || CLM.ModelUtils.generateGUID(),
+        values: makeEntityValues()
+    }
+}
+
+export function makeScoreInput(entityIds?: string[]): CLM.ScoreInput {
+    const fillledEntities = entityIds ?
+        entityIds.map(eid => makeFilledEntity(eid)) : [makeFilledEntity()]
```

Comment:
Minor, but I prefer the results of conditional on separate lines for things more complicated than simple values.

Similar to what you did for `scoreSteps` below:
https://github.com/Microsoft/ConversationLearner-UI/pull/1049/files#diff-daf9ef0c30e15880734bf4a8ef171916R43